app/build.gradle

```diff
@@ -118,6 +118,7 @@ dependencies {
     testImplementation "androidx.room:room-testing:${roomVersion}"
     implementation 'androidx.preference:preference:1.1.0-alpha01'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.google.code.gson:gson:2.8.5"
```

Comment:
Remove

Reply:
type converter can't work without them

Reply:
What is GSON?

Reply:
And why do you need it when the project is using Jackson

Reply:
GSON
A Java serialization/deserialization library to convert Java Objects into JSON and back
Refrences
https://medium.com/@amit.bhandari/storing-java-objects-other-than-primitive-types-in-room-database-11e45f4f6d22

Reply:
> Why do you need it when the project is using Jackson?

Reply:
i need it to convert event topic,subtopic and type model into event model by converting into gson and then converting it back to respective models

Reply:
Why GSON?

Reply:
Why do you need it when the project is using Jackson?

Reply:
@iamareebjamal please read this https://medium.com/@amit.bhandari/storing-java-objects-other-than-primitive-types-in-room-database-11e45f4f6d22 you'll get it

Reply:
i needed event topic models inside event model to save non primitive data type we type converter and gson helps these type converter

Reply:
But, why do you need it when the project is using Jackson?

Reply:
Do you know what Jackson is? I'm not talking about the King of Pop here

Reply:
yes i know Jackson but with Jackson i can't implement the converter part

Reply:
class EventTypeConverter {

     @TypeConverter
    fun toEventSubTopic(json: String): EventType? {
        val type = object : TypeToken<EventType>() {}.type
        return Gson().fromJson(json, type)
    }

     @TypeConverter
    fun toJson(eventSubTopic: EventType?) = Gson().toJson(eventSubTopic) }
here the implementation can't be achieved using jackson

Reply:
Why can't it be done using Jackson?

Reply:
i couldn't find any methods for such thing
https://stackoverflow.com/questions/49064618/typetoken-getparameterized-in-jackson-json

Reply:
I don't think I have a job of pointing to the documentation of a JSON serializing library

http://lmgtfy.com/?q=convert+object+jackson

Reply:
@iamareebjamal i'm sorry and intiallly thought you weren't getting the point of gson/jackson